README.md

```diff
@@ -21,11 +24,21 @@ Currently the following variables are supported:
 
 ### General
 
+* `package_updater_packages` - Required: List of packages to install, update, or remove.
+   Defined as an empty list by default.
+
+* `package_updater_update_cache` - Default: true.  Whether or not to force underlying package manager
+   to check if cache is out of date and re-download if needed.
+
+* `package_updater_state` - Default: latest.  Whether to install (present) update (latest), or
```

Comment:
This should also go away, as we're not giving users this level of control.

Reply:
removed